torch/_functorch/_aot_autograd/collect_metadata_analysis.py

```diff
@@ -225,10 +225,8 @@ def inner(*flat_args):
                     "tensor subclasses"
                 )
 
-            if not isinstance(arg, Tensor):
-                new_arg = arg
-            else:
-                new_arg = from_fun(f_arg)
+            if isinstance(arg, Tensor):
```

Comment:
So a lot of these are finding completely useful no-side effect functions that should be removed it appears